config/routes.rb

```diff
@@ -43,4 +40,20 @@
       resources :group_assignments, path: 'group-assignments'
     end
   end
+
+  namespace :stafftools do
+    mount Sidekiq::Web  => '/sidekiq'
```

Comment:
It feels weird to authenticate the user twice. The controller will always need to lookup `current_user` and any auth logic (like impersonation) would need to be duplicated in the controller and the constraint. You also don't have as much context in the constraint as you do in the controller, leading to authz issues like https://github.com/education/classroom/issues/105. I also feel like doing authn/authz in the controller is idiomatic and putting it elsewhere makes it unclear how/if authentication is working.